app/models/volunteer.rb

```diff
@@ -60,21 +62,21 @@ def supervised_by?(supervisor)
     self.supervisor == supervisor
   end
 
-  def made_contact_with_all_cases_in_days?(num_days = 14)
+  # false if volunteer has any case with no contact in the past 30 days
+  def made_contact_with_all_cases_in_days?(num_days = CONTACT_MADE_IN_DAYS_NUM)
     # TODO this should do the same thing as no_contact_for_two_weeks but for a volunteer
-    total_cases_count = casa_cases.count
+    total_cases_count = casa_cases.size
     return true if total_cases_count.zero?
     current_contact_cases_count = cases_where_contact_made_in_days(num_days).count
     current_contact_cases_count == total_cases_count
   end
 
   private
 
-  def cases_where_contact_made_in_days(num_days = 14)
+  def cases_where_contact_made_in_days(num_days = CONTACT_MADE_IN_DAYS_NUM)
     casa_cases
       .joins(:case_contacts)
-      .where(case_contacts: {contact_made: true})
-      .where("case_contacts.occurred_at > ?", Date.current - num_days.days)
+      .where(case_contacts: {contact_made: true, occurred_at: 14.days.ago.to_date..})
```

Comment:
let's use the `CONTACT_MADE_IN_DAYS_NUM ` variable instead of `14` here

Reply:
🤦